mobile/response.go

```diff
@@ -0,0 +1,64 @@
+package status
```

Comment:
We can either move it to `types.go` or change it to `jsonrpc_response.go`. 

It's an issue that we have so many response types but this one was introduced for bindings that are meant to replace JSON-RPC calls. In status-react, commands like `eth_sendTransaction` are intercepted and a binding is called instead. Thanks to this type of response, the response can be easily returned without further processing.